projects/aca-shared/src/lib/components/document-base-page/document-base-page.spec.ts

```diff
@@ -199,8 +209,18 @@ describe('Info Drawer state', () => {
 
   let store: MockStore<{ app: Partial<AppState> }>;
   let appState: Partial<AppState> = {};
+  let userPreferencesService: any;
```

Comment:
we should avoid using any, please try use for all below services/router following: 
```ts
SpyObj<UserPreferencesService>
```

SpyObj type should resolve problem. When you assign these variables during injection you can do if needed: 
```ts
TestBed.inject(UserPreferencesService) as SpyObj<UserPreferencesService>
```